apps/site/assets/ts/schedule/components/ScheduleFinder.tsx

```diff
@@ -81,19 +83,37 @@ const ScheduleFinder = ({
     });
   };
 
-  return (
-    <div className="schedule-finder">
-      <ScheduleFinderForm
-        onDirectionChange={changeDirection}
-        onOriginChange={changeOrigin}
-        onOriginSelectClick={openOriginModal}
-        onSubmit={openScheduleModal}
-        route={route}
-        selectedDirection={directionId}
-        selectedOrigin={selectedOrigin}
-        stopsByDirection={stops}
-      />
+  const isFerryRoute = routeToModeName(route) === "ferry";
 
+  return (
+    <div
+      className={`${
+        isFerryRoute ? "schedule-finder-vertical" : "schedule-finder"
+      }`}
+    >
+      {isFerryRoute ? (
+        <ScheduleFinderFormVertical
+          onDirectionChange={changeDirection}
+          onOriginChange={changeOrigin}
+          onOriginSelectClick={openOriginModal}
+          onSubmit={openScheduleModal}
+          route={route}
+          selectedDirection={directionId}
+          selectedOrigin={selectedOrigin}
+          stopsByDirection={stops}
+        />
```

Comment:
It might not be necessary to create a separate component/file but instead apply specific styles to the existing one